.github/actions/docker/action.yaml

```diff
@@ -9,7 +9,13 @@ inputs:
     description: Name of image in Docker Hub, like `dashpay/drive`
     required: true
   target:
-    description: "Target stage within Dockerfile: dapi, drive-abci, dashmate-helper, test-suite"
+    description: Target stage within Dockerfile
+    type: choice
+    options:
+      - dapi
+      - drive-abci
+      - dashmate-helper
+      - test-suite
```

Comment:
These params are not supported for action input, they are only for workflow input which is different